src/main/resources/mailTemplates/restorePasswordMailTemplate.txt

```diff
@@ -0,0 +1,39 @@
+<html>
```

Comment:
I suggest this file type and other template files type can be changed to html. The IDE will help with formating. But this is out of scope of this task.

Reply:
Thanks, I'll change it.